_layouts/documentation.html

```diff
@@ -0,0 +1,35 @@
+---
+layout: default
+title: Documentation
+---
+
+<section>
+    <div class="header header-grey">
+      <h2>Documentation</h2>
+      <p>Browse documentation for everything in the Jupyter universe.</p>
```

Comment:
Shorten perhaps:

Browse the Jupyter universe for information. or Browse the Jupyter universe.